src/components/Category.jsx

```diff
@@ -0,0 +1,44 @@
+import StyledHeadline from '../components/StyledHeadline';
+import useStore from '../hooks/useStore';
+
+import StyledContainer from './StyledContainer';
+import VideoThumbnail from './VideoThumbnail';
+
+export default function Category() {
+	const videos = useStore(state => state.videos);
+
+	const allCategories = videos.map(video => {
+		return video.category;
+	});
+
+	let uniqueCategories = [...new Set(allCategories)];
+
+	return uniqueCategories.map(category => {
+		return (
+			<>
+				<StyledContainer variant="exploreWrapper">
+					<StyledHeadline key={category}>{category}</StyledHeadline>
+					<StyledContainer variant="scroll">
+						{videos
+							.filter(video => {
+								return video.category === category;
+							})
+							.map(video => {
+								return (
+									<>
+										<VideoThumbnail
+											key={video.id}
+											YouTubeLink={video.YouTubeLink}
+											videoTitle={video.videoTitle}
+											category={video.category}
+											id={video.id}
+										/>
+									</>
+								);
+							})}
+					</StyledContainer>
+				</StyledContainer>
+			</>
+		);
+	});
```

Comment:
1. You don't need the fragment here.
2. The `key` needs to be on the most outer element - right now that would be the fragment
3. For the video map, you don't need the fragment either
```suggestion
	return uniqueCategories.map(category => {
		return (
				<StyledContainer key={category} variant="exploreWrapper">
					<StyledHeadline>{category}</StyledHeadline>
					<StyledContainer variant="scroll">
						{videos
							.filter(video => {
								return video.category === category;
							})
							.map(video => {
								return (
										<VideoThumbnail
											key={video.id}
											YouTubeLink={video.YouTubeLink}
											videoTitle={video.videoTitle}
											category={video.category}
											id={video.id}
										/>
								);
							})}
					</StyledContainer>
				</StyledContainer>
		);
	});
```

Reply:
@macfrei Done, thanks!